src/components/List/List.tsx

```diff
@@ -1,18 +1,20 @@
 import * as React from 'react';
-import {ListItem} from 'types';
-import Card from '../Card';
-import {Spinner} from '../Spinner';
-import {Container} from './styles';
+import {ListItem} from 'types/types';
+import Card from '../Card/Card';
+import {Spinner} from '../Spinner/Spinner';
+import {ListContainer} from './List.styled';
 
-interface Props {
-    items?: ListItem[];
+interface ListProps {
+    items: ListItem[];
     hasNavigation?: boolean;
-    isLoading: string;
+    isLoading: boolean;
```

Comment:
`boolean` is the correct type of `isLoading`, this was creating an error on this component and where the component is used